check_source.py

```diff
@@ -223,6 +223,15 @@ def check_action_delete(self, request, action):
         except urllib2.HTTPError:
             return None
 
+        # Decline the delete request if there is another delete/submit request against the same package
+        query = "match=state/@name='new'+and+(action/target/@project='{}'+and+action/target/@package='{}')"\
+                "+and+(action/@type='delete'+or+action/@type='submit')".format(action.tgt_project, action.tgt_package)
+        url = osc.core.makeurl(self.apiurl, ['search', 'request'], query)
+        matches = ET.parse(osc.core.http_GET(url)).getroot()
+        if int(matches.attrib['matches']) > 1:
+            self.review_messages['declined'] = "There is pending request to %s/%s in process." % (action.tgt_project, action.tgt_package)
```

Comment:
`There is [a] pending...`

I would include the request ID in the message so people can find it easily.